src/price-formatting/format.spec.ts

```diff
@@ -66,8 +66,11 @@ describe('price-formatting format', () => {
     });
 
     it('requires decimals to be set and positive', () => {
+        // @ts-expect-error

```

Comment:
any plans to fix this?

Reply:
We actually expect TS errors in this cases so no need to fix.